pkg/common/config/option.go

```diff
@@ -56,6 +56,7 @@ type Options struct {
 	DisablePreParseMultipartForm bool
 	StreamRequestBody            bool
 	NoDefaultServerHeader        bool
+	DebugPrintRoute              bool
```

Comment:
- DebugPrintRoute -> DisablePrintRoute
- I think the default should be false and print log

Reply:
> * DebugPrintRoute -> DisablePrintRoute
> * I think the default should be false and print log

Done